### PR TITLE
The string comparision must check for zero for equality.

### PR DESCRIPTION
**Pull request by harishkrupo**
Jira: None
Tests: Board boots to home screen.

Signed-off-by: Harish Krupo <harish.krupo.kps@intel.com>

### CODE REVIEW

File: common/core/gpudevice.cpp [hunk 1]
```diff
@@ -184,9 +184,9 @@ void GpuDevice::DisplayManager::HotPlugEventHandler() {
 
     for (int32_t i = 0; i < ret;) {
       char *event = buffer + i;
-      if (strcmp(event, "DEVTYPE=drm_minor"))
+      if (strcmp(event, "DEVTYPE=drm_minor")==0)
```
Comment (pallavigopal) [inline common/core/gpudevice.cpp]: please add space between the ==

File: common/core/gpudevice.cpp [hunk 2]
```diff
@@ -184,9 +184,9 @@ void GpuDevice::DisplayManager::HotPlugEventHandler() {
...
         drm_event = true;
-      else if (strcmp(event, "HOTPLUG=1"))
+      else if (strcmp(event, "HOTPLUG=1")==0)
```
Comment (pallavigopal) [inline common/core/gpudevice.cpp]: -do- above

Comment (pallavigopal) [inline common/core/gpudevice.cpp]: Please run the clang. we can make !strcmp instead of == also

**Comment by harishkrupo** [comment]
@pallavigopal I have made the required changes. Could you please check?

**Comment by pallavigopal** [comment]
merged thanks

**Pull request merged**